code/controllers/subsystems/records.dm

```diff
@@ -29,6 +32,9 @@
 	excluded_fields["cmp_field"] = "cmp_field"
 	excluded_fields["excluded_fields"] = "excluded_fields"
 
+	InitializeCitizenships()
+	InitializeReligions()
```

Comment:
Maybe put thoes two functions in to here or even put them both in `Initialize ()` for subsystem, as they look more expensive.